sdk/tests/conformance2/transform_feedback/transform_feedback.html

```diff
@@ -114,6 +116,38 @@
     wtu.glErrorShouldBe(gl, gl.NO_ERROR, "there should be no errors");
 }
 
+function runTFBufferBindingTest() {
+    debug("");
+    debug("Testing binding and unbinding transform feedback objects and buffers");
+
+    buf = gl.createBuffer();
+    gl.bindBuffer(gl.TRANSFORM_FEEDBACK_BUFFER, buf);
+    gl.bufferData(gl.TRANSFORM_FEEDBACK_BUFFER, 16, gl.STATIC_DRAW);
+    gl.bindBuffer(gl.TRANSFORM_FEEDBACK_BUFFER, null);
+
+    tf = gl.createTransformFeedback();
+    gl.bindTransformFeedback(gl.TRANSFORM_FEEDBACK, tf);
+    gl.bindBufferBase(gl.TRANSFORM_FEEDBACK_BUFFER, 0, buf);
+    gl.bindTransformFeedback(gl.TRANSFORM_FEEDBACK, null);
+
+    // gl.TRANSFORM_FEEDBACK_BUFFER is part of Transform Feedback objects'
+    // state. See OpenGL ES 3.0.5 Section 6.24.
+    //
+    // Since the TRANSFORM_FEEDBACK was just unbound, there should be nothing
+    // bound.
+    shouldBeNull('gl.getIndexedParameter(gl.TRANSFORM_FEEDBACK_BUFFER_BINDING, 0)');
+
+    // Binding the buffer to the ARRAY_BUFFER binding point should succeed.
+    gl.bindBuffer(gl.ARRAY_BUFFER, buf);
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR, "binding buffer to ARRAY_BUFFER");
```

Comment:
The issue above has nothing to do with relaxing anything. It has to do with actually working period. The test above works as OpenGL is suppose to work

1. create transformfeedback
2. bind transformfeedback
3. bind buffer to TRANSFORM_FEEDBACK
    
    this bind the buffer to a particular index on the TransformFeedback object the same way vertexAttribPointer binds a buffer to an attribute

4.  Unbind the transform feedback

There are now NO buffers bound. There's above attached to a currently unused Transformfeedback. The same way buffers can be attached to unused VertexArrayObjects and the same way textures can be attached to framebuffers. 

Binding to ARRAY_BUFFER shoud not error at this point period.